assets/templates/integrations.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Doppler UI Team</title>
+    <meta name="keywords" content="">
+    <meta name="description" content="Signup layout">
```

Comment:
`content="Signup layout"` --> `content="Integrations layout"` Same below with og tags.

Reply:
This is for internal usage since this very same header is not going to be used anywhere except in the library. I guess it would be a good idea to remove OpenGraph tags as well as unnecessary meta tags.